test/issue-1730/h1-spec.k

```diff
@@ -0,0 +1,5 @@
+module H1-SPEC
+    imports H
+    rule <k> JUMPI 2 bool2Word(X <Int 0) => . </k>
+         <pc> 0 => #if (X <Int 0) #then 2 #else 1 #fi </pc>
```

Comment:
To see the queries sent to the solver:
`kore-exec --log-entries DebugEvaluateCondition`
`kore-exec --solver-transcript kore-exec.smt2`

Reply:
the last lines of `kore-exec.smt2` are 
```
(pop 1 )
(pop 1 )
(push 1 )
(declare-fun <0> () Bool )
(declare-fun <1> () Int )
(declare-fun <2> () Int )
(push 1 )
(assert (and (and <0> (= (= <1> 0 ) true ) ) (not (= 1 <2> ) ) ) )
(check-sat )
(pop 1 )
(pop 1 )
(push 1 )
(declare-fun <0> () Bool )
(declare-fun <1> () Int )
(push 1 )
(assert (and <0> (= (not (= <1> 0 ) ) true ) ) )
(check-sat )
(pop 1 )
(pop 1 )
(push 1 )
(declare-fun <0> () Bool )
(declare-fun <1> () Int )
(declare-fun <2> () Int )
(push 1 )
(assert (and (and <0> (= (not (= <1> 0 ) ) true ) ) (not (= 2 <2> ) ) ) )
(check-sat )
(pop 1 )
(pop 1 )
(exit )
```
I don't see the `X <Int )` condition from `if`. I pasted the whole file here [https://pastebin.com/S8f5uy58](https://pastebin.com/S8f5uy58)

Reply:
Could you also try with `kore-exec --log-entries DebugEvaluateCondition`?

Reply:
I've set `KORE_EXEC_OPTS = --log-level=debug --log-entries DebugEvaluateCondition
`, but when I run `make test-k` nothing gets logged

Reply:
:confused: That should be impossible. The `DebugEvaluateCondition` log entry is called before the solver is invoked. We know that it is sending queries to the solver because of the transcript.